server/services/avatar-service.js

```diff
@@ -8,8 +8,18 @@ module.exports = class UserService {
     };
 
     getAvatar = (avatar_img) => {
-        const avatarPath = path.join(__dirname, `../../avatars/${avatar_img}`);
-        const base64Image = fs.readFileSync(avatarPath, 'base64');
-        return `data:image/jpeg;base64, ${base64Image}`;
+        if (avatar_img) {
+            const avatarPath = path.join(
+                __dirname,
+                `../../avatars/${avatar_img}`
+            );
+            try {
+                const base64Image = fs.readFileSync(avatarPath, 'base64');
+                return `data:image/jpeg;base64, ${base64Image}`;
+            } catch (err) {
+                return '';
+            }
+        }
+        return '';
     };
```

Comment:
Why do you need this? Why you didn't implement this logic as static folder (files)?

Reply:
This logic can't be mixed up with avatar middleware which is only saving avatar in static folder.
Here I checking if avatar exist in my static folder. If it's not then i don't wanna get an error on my UI and only send empty string as src. This will let user to login and upload new img and save it on backend folder.